style.css

```diff
@@ -1053,13 +2327,100 @@ body {
     top: 614px;
   }
 
-  .second__img_back-purse {
+  .second-page__quantity,
+  .third-page__quantity {
     display: none;
   }
 
+  .second__img_purses {
+    width: 63%;
+  }
+
   .second__img_front-purse {
     width: 50%;
   }
+
+  .second-page__article {
+    width: 300px;
+  }
+
+  .second-page__title {
+    font-size: 28px;
+  }
+
+  .second-page__subtitle {
+    font-size: 14px;
+    margin-bottom: 18px;
+  }
+
+  .second-page__description {
+    margin-bottom: 22px;
+  }
+
+  .second-page__color-options {
+    margin-bottom: 28px;
+  }
+
+  .second-page__price {
+    margin-bottom: 28px;
+    font-size: 29px;
+    margin-right: 20px;
+  }
+
+  .second__add-to-cart {
+    width: 100%;
+  }
+
+  .third-page {
+    background: linear-gradient(to right, #EBE3D8 180px, #000 180px, #000 270px, #EBE3D8 270px);
+  }
+
+  .third-page__img {
+    padding: 0 5%;
+  }
+
+  .third-page__wrap {
+    width: 55%;
+    left: 18%;
+  }
+
+  .third-page__article {
+    width: 115%;
+  }
+
+  .third-page__title {
+    font-size: 28px;
+  }
+
+  .third-page__subtitle {
+    font-size: 14px;
+    margin-bottom: 18px; 
+  }
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=6358146)